python/openai/examples/chat.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# or "tensorrt_llm_bls" for TRT-LLM
```

Comment:
I think we should in the final version provide an easy way to alias the model name to be the underlying model name and not trt-llm name

Reply:
Will have a separate PR to refactor which models and names get exposed to user through OpenAI frontend